src/Event/Productions/ProductionRepository.php

```diff
@@ -151,4 +151,33 @@ public function findProductionForEventId(string $eventId): Production
 
         return $production;
     }
+
+    /**
+     * @param string $forEventId
+     * @param ProductionId $inProductionId
+     *
+     * @return Tuple[]
+     * @throws EntityNotFoundException
+     */
+    public function findTuples(string $forEventId, ProductionId $inProductionId): array
+    {
+        $results = $this->getConnection()->fetchAll(
+            'SELECT * FROM productions WHERE production_id = :productionId AND event_id <> :eventId',
```

Comment:
Implementation detail: I prefer the `!=` syntax over `<>`, especially for non-numerical values like uuids